### PR TITLE
forgot password

### PR DESCRIPTION
**Pull request by rbsowmi**

### CODE REVIEW

File: app/components/forgot-password.hbs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<div class="container">
+
+    <h1 class="strips">NextGen</h1>
+
```
Comment (ajalkrishna) [inline app/components/forgot-password.hbs:18]: remove unwanted spaces

File: app/components/forgot-password.hbs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    <label class="email">Email</label>
+    <input type="text" class="input" required>
+    <button class="button">Continue</button>
+
+
+    <p5 class="signup">Don't have an account?<LinkTo @route="signup">Sign up</LinkTo>
```
Comment (ajalkrishna) [inline app/components/forgot-password.hbs]: Why added LinkTo for this button?

File: app/components/forgot-password.hbs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    </p5>
+</div>
+<div class="terms">
```
Comment (ajalkrishna) [inline app/components/forgot-password.hbs]: Better to use icons..instead of unicodes

File: app/components/login-form.hbs [hunk 4]
```diff
@@ -12,5 +12,6 @@
   <div>
     <small class="text-danger">{{this.errorMessage}}</small>
   </div>
-  <button type="submit" class="btn btn-primary" {{on "click" this.signIn}}>Sign In</button>
+  <button  type="submit" class="btn btn-primary" {{on "click" this.signIn}}>Sign In</button>
+  <LinkTo @route="forgot-password">forgotpassword</LinkTo>
```
Comment (ajalkrishna) [inline app/components/login-form.hbs]: Check whether the UI text is fine or not with Vaishnavi

File: app/components/login-form.js [hunk 5]
```diff
@@ -4,55 +4,49 @@ import { tracked } from '@glimmer/tracking';
...
-    @tracked password;
-    @tracked errorMessage
+  @service router;
+  @tracked emailError;
+  @tracked passwordError;
+  @tracked email;
+  @tracked password;
+  @tracked errorMessage;
 
-    validateEmail(email) {
```
Comment (ajalkrishna) [inline app/components/login-form.js:14]: Why this much changes in this file?

File: app/router.js [hunk 6]
```diff
@@ -8,5 +8,7 @@ export default class Router extends EmberRouter {
 
 Router.map(function () {
   this.route('login', { path: '/' });
+  this.route('forgot-password', { path: '/forgotpassword' });
```
Comment (ajalkrishna) [inline app/router.js]: Why to have separate path defined?

File: app/router.js [hunk 7]
```diff
@@ -8,5 +8,7 @@ export default class Router extends EmberRouter {
...
+  this.route('signup');
```
Comment (ajalkrishna) [inline app/router.js]: Do we need a signup route now?

File: app/styles/forgot-password.css [hunk 8]
```diff
@@ -0,0 +1,68 @@
+
```
Comment (ajalkrishna) [inline app/styles/forgot-password.css]: Need updates in the margin property

File: app/templates/forgot-password.hbs [hunk 9]
```diff
@@ -0,0 +1,4 @@
+<div class="col-md-12">
```
Comment (ajalkrishna) [inline app/templates/forgot-password.hbs]: What does this mean?

File: app/components/forgot-password.hbs (hunk 1 above)
Comment (rbsowmi) [reply app/components/forgot-password.hbs:18]: yeah i removed the space

File: app/components/forgot-password.hbs (hunk 2 above)
Comment (rbsowmi) [reply app/components/forgot-password.hbs]: i asked to vaishnavi

File: app/components/forgot-password.hbs (hunk 3 above)
Comment (rbsowmi) [reply app/components/forgot-password.hbs]: okay

File: app/components/login-form.hbs (hunk 4 above)
Comment (rbsowmi) [reply app/components/login-form.hbs]: its fine only that by mistakes

File: app/components/login-form.js (hunk 5 above)
Comment (rbsowmi) [reply app/components/login-form.js:14]: by mistakes i will correct it hereafters

File: app/router.js (hunk 6 above)
Comment (rbsowmi) [reply app/router.js]: they does not gave any forgorpassword in login so i added i asked charitha and rahul for that after only they said to add

File: app/router.js (hunk 7 above)
Comment (rbsowmi) [reply app/router.js]: that one i asked to vaishanvi

File: app/templates/forgot-password.hbs (hunk 9 above)
Comment (rbsowmi) [reply app/templates/forgot-password.hbs]: that 6 i will change

File: app/styles/forgot-password.css (hunk 8 above)
Comment (rbsowmi) [reply app/styles/forgot-password.css]: okay

**Pull request merged**